dashboard/common/db_util.py

```diff
@@ -195,3 +196,51 @@ def get_canvas_data_date() -> Union[datetime, None]:
     except Exception:
         logger.info("Value could not be found from metadata", exc_info = True)
     return None
+
+def map_dict_to_query_job_config(param_dict: Dict[str, Any]) -> bigquery.QueryJobConfig:
```

Comment:
Cannot find the use of this function in the PR, though the cron runs successfully with this PR updates.